### PR TITLE
[WEBSITE] Updates for GKE Policy Library Enhancements

### PR DESCRIPTION
**Pull request by kevensen**
This PR supports:
* terraform-google-forseti PR https://github.com/forseti-security/terraform-google-forseti/pull/356
* helm-charts PR https://github.com/forseti-security/helm-charts/pull/46

- [x] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)

### CODE REVIEW

File: _docs/_latest/setup/forseti-on-gke.md [hunk 1]
```diff
@@ -123,27 +124,60 @@ module "forseti-on-gke-with-config-validator" {
...
+configValidator.policyLibrary.gitSync.enabled: true
 ```
+#### Policy Library in a GCS bucket
+In the Helm example above, the following variables are required in the user defined *values.yaml* file.
+
+```yaml
+# configValidator.policyLibrary.bucket is the GCS
+# storage bucket containing the policy-library.
+# This overrides policyLibrary.respositoryURL.
+# Ommit the gs://.
```
Comment (gkowalski-google) [inline _docs/_latest/setup/forseti-on-gke.md]: Typo, you really like this word `ommit` :)

File: _docs/_latest/setup/forseti-on-gke.md [hunk 2]
```diff
@@ -99,17 +99,18 @@ helm upgrade -i forseti forseti-security/forseti-security \
     --values=forseti-values.yaml
 ```
 
-### Deploying with config-validator on-GKE
+## Deploying with config-validator on-GKE
 
 The config-validator in Forseti on-GKE obtains policies from a policy-library in a Git repository via SSH.  The pre-requisites for this are as follows.
```
Comment (gkowalski-google) [inline _docs/_latest/setup/forseti-on-gke.md]: I think this section should be updated to say that we support Git and GCS. The pre-requisites for Git can be moved into the Policy Library in a Git Repository section, and add the pre-reqs for GCS. I think the only pre-req for GCS is the user will need to create the `policy-library` directory in the server GCS bucket.

File: _docs/_latest/setup/forseti-on-gke.md [hunk 3]
```diff
@@ -123,27 +124,60 @@ module "forseti-on-gke-with-config-validator" {
...
-configValidator: true
+# configValidator.enabled sets whether or not to deploy config-validator
+configValidator.enabled: true
 
-# gitSyncPrivateSSHKey is the private OpenSSH key generated to allow the git-sync to clone the policy library repository.
-gitSyncPrivateSSHKey: ""
+# configValidator.policyLibrary.gitSync.privateSSHKey
+# is the private OpenSSH key generated to allow the 
+# git-sync to clone the policy library repository.
+configValidator.policyLibrary.gitSync.privateSSHKey: ""
```
Comment (gkowalski-google) [inline _docs/_latest/setup/forseti-on-gke.md]: I believe there is an issue with the recent GCS changes made to the Terraform and Helm. In doing some testing of a new feature, I am having a hard time getting the helm release to successfully deploy the cluster (mainly CV pod will not start). Is there an issue here where Terraform is still providing the `configValidator` prefix to these values when the CV values has been moved into it's own chart? https://github.com/forseti-security/helm-charts/blob/master/charts/config-validator/values.yaml

File: _docs/_latest/setup/forseti-on-gke.md (hunk 1 above)
Comment (kevensen) [reply _docs/_latest/setup/forseti-on-gke.md]: ack

**Pull request merged**